pkg/cstor/pv_operation.go

```diff
@@ -148,3 +149,12 @@ func generateClonePVName() (string, error) {
 
 	return PvClonePrefix + nuuid.String(), nil
 }
+
+// isCSIPv returns true if given PV is created by cstor CSI driver
+func isCSIPv(pv v1.PersistentVolume) bool {
```

Comment:
this can be isCSIPV.. change it if only making any must commit

Reply:
I will fix this next PR.